src/vs/base/browser/ui/actionbar/actionbar.css

```diff
@@ -71,14 +71,12 @@
 	display: block;
 	border-bottom: 1px solid #bbb;
 	padding-top: 1px;
-	margin-left: .8em;
-	margin-right: .8em;
 }
 
 .monaco-action-bar .action-item .action-label.separator {
 	width: 1px;
 	height: 16px;
-	margin: 5px 4px !important;
+	margin: 5px 0px !important;
```

Comment:
this impacts non-menus too

Reply:
![image](https://user-images.githubusercontent.com/6561887/168662171-8d190aae-2a80-492b-a2da-a6154f7e3078.png)

Reply:
Reverted since the more specific rule is respected in menu.ts